### ISSUE TITLE
Move logo and clock randomly in the screensaver to prevent burn-ins on OLEDs

### ISSUE THREAD

**Issue by EmberLightVFX** [jellyfin/jellyfin-androidtv#1888]
### Describe the feature you'd like

Moving the Jellyfin logo and the clock to different positions when the screensaver is active would help prevent burn-ins for OLED TVs. Ether they could jump to different corners or maybe be possible to dissable them in the settings.

**Comment by kingp0dd** [comment]
+1

**Comment by kingp0dd** [comment]
is there a chance that will make it to the new roadmap? :)

**Comment by nielsvanvelzen** [comment]
Yes this is definitely something we should address, added to the roadmap.

**Comment by Hukuma1** [comment]
The new screensaver is great, but would love the option to disable static elements. (clock + jellyfin logo)

Had a scare because I forgot to turn off the TV the other night. D:

**Comment by nielsvanvelzen** [comment]
Starting with version 0.19 of the app the screensaver will no longer show the Jellyfin logo which fixes this issue.

**Comment by Passific** [comment]
By the PR title and content, I guess the clock is staying put.
That does not completely close the issue then.

**Comment by kingp0dd** [comment]
> By the PR title and content, I guess the clock is staying put. That does not completely close the issue then.

Your are right

**Comment by Zark247** [comment]
You can disable the clock from the gear icon (Preferences):
Customization > Show Clock : Never
